src/actions/remove.js

```diff
@@ -42,6 +43,25 @@ function addMetadata(userData) {
     .then((metadata) => [userData, metadata]);
 }
 
+async function removeOrganizationUser(userId) {
```

Comment:
Maybe we should avoid context binding, this will be a bit faster? 😄 

```javascript
async function removeOrganizationUser(service, userId) {}
```
And then:
```javascript
await removeOrganizationUser(this, userId)
```